README.md

```diff
@@ -4,9 +4,21 @@
 
 [SMS module](https://github.com/johnsonandjohnson/openmrs-distro-cfl/wiki/Admin-SMS) to enable track and sending SMS to OpenMRS users.
 
+[Messsages module](https://github.com/johnsonandjohnson/openmrs-distro-cfl/wiki/Admin-Messages) to enable configuration and sending Message services to OpenMRS users
+
 ## Running this code
 
 ```sh
 yarn  # to install dependencies
-yarn start --sources 'packages/esm-sms-app' # to run a specific application
+
+yarn start --sources 'packages/esm-sms-app' / 'packages/esm-messages-app' # to run a specific application
```

Comment:
I actually think the existing example is much clearer as it's explaining the syntax. We don't need to add _every_ option to the README.